shlr/Makefile

```diff
@@ -34,7 +34,7 @@ TS_TIP=f049ba350f3f6019ce9a1cbb0975ebd154ef7ad3
 # NOTE: when you update SHELLPARSER_TIP or SHELLPARSER_BRA, also update them in shlr/meson.build
 SHELLPARSER_URL=https://github.com/ret2libc/radare2-shell-parser.git
 SHELLPARSER_BRA=master
-SHELLPARSER_TIP=167f8beca060bfab89f4b8e3ef47ffbb071437f2
+SHELLPARSER_TIP=3d82cad9d865cb6e65364f66f038d1d1d4d8818a
```

Comment:
??? what happened here?

Reply:
Everything happened here.

Reply:
im rebasing to master because of the conflicts and wtf is going on

Reply:
too many changes in too many files, so as soon as there is something in master, you have conflicts :)